RadarSDK/RadarNotificationHelper.m

```diff
@@ -320,10 +320,16 @@ + (void)getNotificationDiffWithCompletionHandler:(void (^)(NSArray *notification
     [notificationCenter getPendingNotificationRequestsWithCompletionHandler:^(NSArray<UNNotificationRequest *> *requests) {
         NSMutableArray *currentNotifications = [NSMutableArray new];
         
+        
+        [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:[NSString stringWithFormat:@"NotificationDiff processing difference between registered and pending notifications"]];
+        for (NSDictionary* n in registeredNotifications) {
```

Comment:
Ideally we get away from logs in for loops. Better to have a single log line at the end of the for loop. Makes debugging much easier